src/components/Preferences/Availability.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+    Box,
+    Checkbox,
+    HStack,
+    Flex,
+    Select,
+    Stack,
+    Text,
+} from "@chakra-ui/react";
+import { useState } from "react";
+
+const Availability = () => {
+    const [isReliefDisabled, setIsReliefDisabled] = useState(true);
+    const [isSabbaticalDisabled, setIsSabbaticalDisabled] = useState(true);
+
+    return (
+        <Box>
+            <Stack spacing={5} direction="column">
+                <HStack spacing={5}>
+                    <Text>Please check desired semester(s) off:</Text>
```

Comment:
wording may be confusing

Reply:
In the mockup we had "select desired teaching semesters" but Amanda told me on Tuesday it should be the opposite so that's why I switched it. We can discuss if this is still a confusing way to say that.